packages/ethers/src/index.ts

```diff
@@ -29,6 +29,11 @@ import {
 import type { TurnkeyBrowserClient } from "@turnkey/sdk-browser";
 import type { TurnkeyServerClient } from "@turnkey/sdk-server";
 
+type TPayloadEncoding =
```

Comment:
we can actually use some of our exported types from `http` here. 

import { 
  type TurnkeyApiTypes
} from "@turnkey/http";

...

type TPayloadEncoding = TurnkeyApiTypes["v1PayloadEncoding"];
```